src/new-apps/components/AppListPage/utils.ts

```diff
@@ -0,0 +1,47 @@
+import { AppListItemFragment } from "@saleor/graphql";
+import { GetV2SaleorAppsResponse } from "@saleor/new-apps/marketplace.types";
+import { InstalledApp } from "@saleor/new-apps/types";
+
+import { AppListPageSections } from "./types";
+
+export const resolveSectionsAvailability = ({
+  installedApps,
+  installableMarketplaceApps,
+  comingSoonMarketplaceApps,
+}: AppListPageSections) => ({
+  installed: !installedApps || !!installedApps.length,
+  all: !installableMarketplaceApps || !!installableMarketplaceApps.length,
+  comingSoon: !comingSoonMarketplaceApps || !!comingSoonMarketplaceApps.length,
+});
+
+const isAppExternal = (
+  manifestUrl: string,
+  installableMarketplaceApps?: GetV2SaleorAppsResponse.ReleasedSaleorApp[],
+) => installableMarketplaceApps?.every(app => app.manifestUrl !== manifestUrl);
+
+export const getVerifiedInstalledApps = (
+  installedApps?: AppListItemFragment[],
+  installableMarketplaceApps?: GetV2SaleorAppsResponse.ReleasedSaleorApp[],
+): InstalledApp[] =>
```

Comment:
Sure, it's kinda important to slowly but consequently and finally get rid of these null errors mess. Fixed.
And question, maybe can we introduce local tsconfigs as following so it would be easier to migrate?:
```json
{
  "extends": "../../tsconfig.json",
  "compilerOptions": {
    "strictNullChecks": true
  }
}
```
And once migration is complete we could remove these files?

Reply:
cc @andrzejewsky

Reply:
I think local tsconfigs require all imports to also follow strict null checks so it's pretty difficult to do, but you can try - if it works it would be nice.

Reply:
Ok, added.